fuzz/fuzz.py

```diff
@@ -337,7 +337,7 @@ def run_fuzz():
           set([msg.command for msg in message_objects]))
 
     num_message = 100
-    num_SIGINT = 10
+    # INT = 10
```

Comment:
It's used in the commented line below. Maybe the name of the variable should not change.

Reply:
Yes, I guess so. It's my mistake.